acceptance/run.sh

```diff
@@ -3,4 +3,4 @@
 cd $(dirname $0)/..
 build/builder.sh make install
 set -x
-go test -v -tags acceptance ./acceptance ${GOFLAGS} -run "${TESTS:-.*}" -timeout ${TESTTIMEOUT:-1m} ${TESTFLAGS}
```

Comment:
Yes. All the acceptance tests are unfortunately quite slow; we're already up to 55s on my machine.